File: JavaProgLang/ArrayJava/LinearSearch.java
import java.util.*;

public class LinearSearch {
    public static void LinSearch(byte arr[], byte key) {
        boolean found=false;
        for (byte i = 0; i < arr.length; i++) {
            if (arr[i] == key) {
                System.out.println("The key is at index of "+i);
                found=true;
                break;
            } 
        }
        if(!found){
            System.out.println("key is not found. Please enter the correct key");
        }

    }

    public static void main(String args[]) {
        System.out.println("Enter the size of an array: ");
        Scanner s = new Scanner(System.in);
        byte size = s.nextByte();
        System.out.println("Enter the elements into the array");
        byte arr[] = new byte[size];
        for (byte i = 0; i <arr.length; i++) {
            arr[i] = s.nextByte();
        }
        System.out.println("Enter the key to search in array: ");
        byte key = s.nextByte();
        LinSearch(arr, key);

    }
}

File: C/Array/LinearSearch.c
#include<stdio.h>
#include<stdbool.h>
void LinSearch(short arr[],short key,int size){
    bool found=false;
    for(short i=0;i<size;i++){
        if(arr[i]==key){

        printf("key is found at index %hd",i);
        found=true;
        }
    }
    if(!found){
        printf("\nKey is not found. Please Enter the correct key");
    }
}
int main(){
    printf("Enter the size of an array: ");
    short size;
    scanf("%hd",&size);
    short arr[size];
    printf("\nEnter the data into the array: ");
    for(short i=0;i<size;i++){
        scanf("%hd",&arr[i]);
    }
    printf("\nEnter the key to search: ");
    short key;
    scanf("%hd",&key);
    LinSearch(arr,key,size);
}